src/util/formChangeCount.tsx

```diff
@@ -0,0 +1,143 @@
+import {
```

Comment:
I think this file's functions can benefit from some unit tests. Perhaps a follow up task?

Reply:
I added some tests to it.